numba/cuda/libdevice.py

```diff
@@ -165,3 +167,58 @@ def core(context, builder, sig, args):
 for (ty, intrin) in ((types.float64, '__nv_modf',),
                      (types.float32, '__nv_modff',)):
     lower(math.modf, ty)(modf_implement(intrin, ty))
+
+
+# Complex power implementations - translations of _Py_c_pow from CPython
+# 3.7.5.
```

Comment:
Please add an exact reference with SHA to the implementation. Example:
https://github.com/numba/numba/blob/626b40eddc9a29a1e89bfb6b8706e4f390cfcc4a/numba/cpython/hashing.py#L30-L31